sample_2011/geography.csv

```diff
@@ -0,0 +1,11 @@
+Country11CD,Country11NM,Country11NMW,Region11CD,Region11NM,Region11NMW
```

Comment:
I don't think it really matters, but I'd have expected to see the `Region` columns before the `Country` columns.